Kernel/Arch/aarch64/PSCI.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2025, the SerenityOS developers
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+namespace Kernel::PSCI {
+
```

Comment:
Adding a spec reference to https://developer.arm.com/documentation/den0022/latest/
would be nice.